### DISCUSSION TITLE
Unable to install python packages

### DISCUSSION THREAD

**Discussion by get2aakash** [semaphoreui/semaphore#2086] [Q&A]
Getting error when trying to install python package using requirements.txt ..Semaphore container does not start..

[root@gew3pfgtansbl1 automation]# cat requirements.txt
--trusted-host pypi.org
--trusted-host files.pythonhosted.org

pretty_errors

Error:
WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLEOFError(8, '[SSL: UNEXPECTED_EOF_WHILE_READING] EOF occurred in violation of protocol (_ssl.c:1006)'))': /simple/pretty-errors/
Could not fetch URL https://pypi.org/simple/pretty-errors/: There was a problem confirming the ssl certificate: HTTPSConnectionPool(host='pypi.org', port=443): Max retries exceeded with url: /simple/pretty-errors/ (Caused by SSLError(SSLEOFError(8, '[SSL: UNEXPECTED_EOF_WHILE_READING] EOF occurred in violation of protocol (_ssl.c:1006)'))) - skipping
ERROR: Could not find a version that satisfies the requirement pretty_errors (from versions: none)
ERROR: No matching distribution found for pretty_errors

**Comment by tboerger** [comment]
Maybe drop the trusted host settings? Without that it was working fine so far. There is just a pending PR to install the packages into venv and not into home directory of the Semaphore user.

**Comment by fiftin** [comment]
Hi @get2aakash could you try latest beta? 

[![](https://img.shields.io/badge/semaphore_2.10.4_(beta)_container_configurator-skyblue?style=for-the-badge&logo=docker)](https://semaphoreui.com/install/docker/2_10_4-beta/)

**Reply by srirag21** [reply]
Works for me, thanks!
